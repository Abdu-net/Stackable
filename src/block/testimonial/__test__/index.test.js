import { edit, save } from '../index'

describe( 'Testimonial', () => {
	test( 'block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				href: 'http://www.gambit.ph/',
				mediaID: '1',
				mediaURL: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				testimonialTitle: 'Ben Adams',
				position: 'Founder',
				body: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				titleColor: '#000000',
				posColor: '#000000',
				bodyTextColor: '#000000',
				iconColor: '#000000',
				columns: '1',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '2 column block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				href: 'http://www.gambit.ph/',
				hrefTwo: 'http://www.gambit.ph/',
				mediaID: '1',
				mediaIDTwo: '2',
				mediaURL: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				mediaURLTwo: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				testimonialTitle: 'Ben Adams',
				testimonialTitleTwo: 'Alex Johnson',
				position: 'Founder',
				positionTwo: 'Editor',
				body: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				bodyTwo: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				titleColor: '#000000',
				posColor: '#000000',
				bodyTextColor: '#000000',
				iconColor: '#000000',
				columns: '2',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '3 column block edit matches snapshots', () => {
		const wrapper = edit( {
			isSelected: false,
			attributes: {
				href: 'http://www.gambit.ph/',
				hrefTwo: 'http://www.gambit.ph/',
				hrefThree: 'http://www.gambit.ph/',
				mediaID: '1',
				mediaIDTwo: '2',
				mediaIDThree: '3',
				mediaURL: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				mediaURLTwo: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				mediaURLThree: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				testimonialTitle: 'Ben Adams',
				testimonialTitleTwo: 'Alex Johnson',
				testimonialTitleThree: 'Sammy Simpson',
				position: 'Founder',
				positionTwo: 'Editor',
				positionThree: 'Programmer',
				body: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				bodyTwo: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				bodyThree: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				titleColor: '#000000',
				posColor: '#000000',
				bodyTextColor: '#000000',
				iconColor: '#000000',
				columns: '3',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( 'block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				href: 'http://www.gambit.ph/',
				mediaID: '1',
				mediaURL: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				testimonialTitle: 'Ben Adams',
				position: 'Founder',
				body: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				titleColor: '#000000',
				posColor: '#000000',
				bodyTextColor: '#000000',
				iconColor: '#000000',
				columns: '1',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '2 column block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				href: 'http://www.gambit.ph/',
				hrefTwo: 'http://www.gambit.ph/',
				mediaID: '1',
				mediaIDTwo: '2',
				mediaURL: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				mediaURLTwo: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				testimonialTitle: 'Ben Adams',
				testimonialTitleTwo: 'Alex Johnson',
				position: 'Founder',
				positionTwo: 'Editor',
				body: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				bodyTwo: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				titleColor: '#000000',
				posColor: '#000000',
				bodyTextColor: '#000000',
				iconColor: '#000000',
				columns: '2',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )

	test( '3 column block save matches snapshots', () => {
		const wrapper = save( {
			isSelected: false,
			attributes: {
				href: 'http://www.gambit.ph/',
				hrefTwo: 'http://www.gambit.ph/',
				hrefThree: 'http://www.gambit.ph/',
				mediaID: '1',
				mediaIDTwo: '2',
				mediaIDThree: '3',
				mediaURL: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				mediaURLTwo: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				mediaURLThree: 'https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80',
				testimonialTitle: 'Ben Adams',
				testimonialTitleTwo: 'Alex Johnson',
				testimonialTitleThree: 'Sammy Simpson',
				position: 'Founder',
				positionTwo: 'Editor',
				positionThree: 'Programmer',
				body: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				bodyTwo: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				bodyThree: 'Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage.',
				titleColor: '#000000',
				posColor: '#000000',
				bodyTextColor: '#000000',
				iconColor: '#000000',
				columns: '3',
			},

		} )

		expect( wrapper ).toMatchSnapshot()
	} )
} )
