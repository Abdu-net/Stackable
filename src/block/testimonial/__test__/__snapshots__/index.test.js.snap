// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testimonial block edit matches snapshots 1`] = `
<Fragment>
  <InspectorControls>
    <PanelBody>
      <SelectControl
        label="Column Number"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "One Column",
              "value": "1",
            },
            Object {
              "label": "Two Column",
              "value": "2",
            },
            Object {
              "label": "Three Column",
              "value": "3",
            },
          ]
        }
        value="1"
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Title Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Position Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Body Text Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Icon Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Color Settings"
    />
  </InspectorControls>
  <div
    className="ugb-testimonial column-1"
  >
    <div
      className="ugb-testimonial-column-one"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value="1"
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value="Ben Adams"
      />
      <RichText
        className="ugb-testimonial-position"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Founder"
      />
      <RichText
        className="ugb-testimonial-body"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage."
      />
    </div>
    <div
      className="ugb-testimonial-column-two"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value={undefined}
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value={undefined}
      />
      <RichText
        className="ugb-testimonial-position-two"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <RichText
        className="ugb-testimonial-body-two"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
    </div>
    <div
      className="ugb-testimonial-column-three"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value={undefined}
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value={undefined}
      />
      <RichText
        className="ugb-testimonial-position-three"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <RichText
        className="ugb-testimonial-body-three"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
    </div>
  </div>
</Fragment>
`;

exports[`Testimonial block edit matches snapshots 2`] = `
<Fragment>
  <InspectorControls>
    <PanelBody>
      <SelectControl
        label="Column Number"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "One Column",
              "value": "1",
            },
            Object {
              "label": "Two Column",
              "value": "2",
            },
            Object {
              "label": "Three Column",
              "value": "3",
            },
          ]
        }
        value="2"
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Title Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Position Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Body Text Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Icon Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Color Settings"
    />
  </InspectorControls>
  <div
    className="ugb-testimonial column-2"
  >
    <div
      className="ugb-testimonial-column-one"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value="1"
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value="Ben Adams"
      />
      <RichText
        className="ugb-testimonial-position"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Founder"
      />
      <RichText
        className="ugb-testimonial-body"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage."
      />
    </div>
    <div
      className="ugb-testimonial-column-two"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value="2"
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value="Alex Johnson"
      />
      <RichText
        className="ugb-testimonial-position-two"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Editor"
      />
      <RichText
        className="ugb-testimonial-body-two"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage."
      />
    </div>
    <div
      className="ugb-testimonial-column-three"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value={undefined}
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value={undefined}
      />
      <RichText
        className="ugb-testimonial-position-three"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
      <RichText
        className="ugb-testimonial-body-three"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value={undefined}
      />
    </div>
  </div>
</Fragment>
`;

exports[`Testimonial block edit matches snapshots 3`] = `
<Fragment>
  <InspectorControls>
    <PanelBody>
      <SelectControl
        label="Column Number"
        onChange={[Function]}
        options={
          Array [
            Object {
              "label": "One Column",
              "value": "1",
            },
            Object {
              "label": "Two Column",
              "value": "2",
            },
            Object {
              "label": "Three Column",
              "value": "3",
            },
          ]
        }
        value="3"
      />
    </PanelBody>
    <PanelColorSettings
      colorSettings={
        Array [
          Object {
            "label": "Title Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Position Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Body Text Color",
            "onChange": [Function],
            "value": "#000000",
          },
          Object {
            "label": "Icon Color",
            "onChange": [Function],
            "value": "#000000",
          },
        ]
      }
      initialOpen={false}
      title="Color Settings"
    />
  </InspectorControls>
  <div
    className="ugb-testimonial column-3"
  >
    <div
      className="ugb-testimonial-column-one"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value="1"
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value="Ben Adams"
      />
      <RichText
        className="ugb-testimonial-position"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Founder"
      />
      <RichText
        className="ugb-testimonial-body"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage."
      />
    </div>
    <div
      className="ugb-testimonial-column-two"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value="2"
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value="Alex Johnson"
      />
      <RichText
        className="ugb-testimonial-position-two"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Editor"
      />
      <RichText
        className="ugb-testimonial-body-two"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage."
      />
    </div>
    <div
      className="ugb-testimonial-column-three"
    >
      <div>
        <MediaUpload
          allowedTypes={
            Array [
              "image",
            ]
          }
          onSelect={[Function]}
          render={[Function]}
          value="3"
        />
      </div>
      <RichText
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="h4"
        value="Sammy Simpson"
      />
      <RichText
        className="ugb-testimonial-position-three"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Programmer"
      />
      <RichText
        className="ugb-testimonial-body-three"
        keepPlaceholderOnFocus={true}
        onChange={[Function]}
        style={
          Object {
            "color": "#000000",
          }
        }
        tagName="p"
        value="Stackable: Ultimate Blocks from Gutenberg has all the blocks I need to make a great webpage."
      />
    </div>
  </div>
</Fragment>
`;

exports[`Testimonial block save matches snapshots 1`] = `
<div
  className="ugb-testimonial column-1"
>
  <div
    className="ugb-testimonial-column-one"
  >
    <div
      className="testimonial-image"
      data-src="https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80"
      style={
        Object {
          "backgroundImage": "url(https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80)",
        }
      }
    />
  </div>
</div>
`;

exports[`Testimonial block save matches snapshots 2`] = `
<div
  className="ugb-testimonial column-2"
>
  <div
    className="ugb-testimonial-column-one"
  >
    <div
      className="testimonial-image"
      data-src="https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80"
      style={
        Object {
          "backgroundImage": "url(https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80)",
        }
      }
    />
  </div>
  <div
    className="ugb-testimonial-column-two"
  >
    <div
      className="testimonial-image"
      data-src="https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80"
      style={
        Object {
          "backgroundImage": "url(https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80)",
        }
      }
    />
  </div>
</div>
`;

exports[`Testimonial block save matches snapshots 3`] = `
<div
  className="ugb-testimonial column-3"
>
  <div
    className="ugb-testimonial-column-one"
  >
    <div
      className="testimonial-image"
      data-src="https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80"
      style={
        Object {
          "backgroundImage": "url(https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80)",
        }
      }
    />
  </div>
  <div
    className="ugb-testimonial-column-two"
  >
    <div
      className="testimonial-image"
      data-src="https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80"
      style={
        Object {
          "backgroundImage": "url(https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80)",
        }
      }
    />
  </div>
  <div
    className="ugb-testimonial-column-three"
  >
    <div
      className="testimonial-image"
      data-src="https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80"
      style={
        Object {
          "backgroundImage": "url(https://images.unsplash.com/photo-1522198734915-76c764a8454d?ixlib=rb-0.3.5&ixid=eyJhcHBfaWQiOjEyMDd9&s=7892e6f709c6e46512bb4b08ce3d1a3c&auto=format&fit=crop&w=1050&q=80)",
        }
      }
    />
  </div>
</div>
`;
